nucliadb/src/nucliadb/search/search/query_parser/parser.py

```diff
@@ -123,9 +122,6 @@ def _parse_reranker(self) -> Reranker:
             if self.item.reranker == search_models.RerankerName.NOOP:
                 reranking = NoopReranker()
 
-            elif self.item.reranker == search_models.RerankerName.MULTI_MATCH_BOOSTER:
```

Comment:
I think you might be able to remove the constant definition (RerankerName.MULTI_MATCH_BOOSTER) as well.